week-2/F-logical-operators/exercise2.js

```diff
@@ -5,17 +5,31 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(number) {
+  if (number < 0) return true;
```

Comment:
This is technically correct, but 99% of the time when we are using `if` statements we also want to use curly braces (`{` and `}`).